.github/workflows/publish-image-dockerhub.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish image to DockerHub
+on:
+  push:
+    branches:
+      - cere-master
```

Comment:
Agreed to use single branch and tags as a trigger for publishing to public registry.